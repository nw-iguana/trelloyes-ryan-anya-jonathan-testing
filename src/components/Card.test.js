import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';
import renderer from 'react-test-renderer';


describe('tests for displaying Card component',()=> {
    it('renders a card without crashing', ()=> {
        const div = document.createElement('div');
        ReactDOM.render(<Card title="card-title" content="card content"/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    
    it('renders the UI as expected', ()=> {
        const tree = renderer
            .create(<Card title="card-title" content="card content" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
});

