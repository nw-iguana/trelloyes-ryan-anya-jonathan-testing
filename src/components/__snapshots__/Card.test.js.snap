// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for displaying Card component renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
     
    card-title
     
  </h3>
  <p>
     
    card content
     
  </p>
</div>
`;
